core/src/apps/common/signverify.py

```diff
@@ -33,3 +36,24 @@ def split_message(message: bytes) -> List[str]:
         m = "hex(%s)" % hexlify(message).decode()
         words = [m]
     return words
+
+
+async def require_confirm_sign_message(
+    ctx: wire.Context, header: str, message: bytes
+) -> None:
+    message = split_message(message)
```

Comment:
why is `split_message` used here?
is there a performance advantage for the renderer to get an array of words, as opposed to the original string?

Reply:
It was used before and this way with `new_lines=False` it looks like this:

![now](https://user-images.githubusercontent.com/1835345/75465353-ce501300-5988-11ea-972d-4ee331c493c2.png)

And without:

![now2](https://user-images.githubusercontent.com/1835345/75465375-d5772100-5988-11ea-85f4-b1a754af7fff.png)

I realize we have actually agreed on this second option. I have probably pushed the wrong code :/. Will fix

Reply:
So the problem with this that it might be quite ugly sometimes:

![image](https://user-images.githubusercontent.com/1835345/75481420-517e6280-59a3-11ea-8754-784c7ca66a73.png)

Reply:
Coming back to this, it does not seem this (as in last comment) is any better. So I am leaving it with `new_lines=False` and splitting for now.